.github/workflows/test.yml

```diff
@@ -73,11 +73,11 @@ jobs:
         shard: ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]
     timeout-minutes: 20
     steps:
-        - uses: alexellis/setup-arkade@master
-        - name: Install vmmeter
-           run: |
-            sudo -E arkade oci install ghcr.io/openfaasltd/vmmeter:latest --path /usr/local/bin/
-        - uses: self-actuated/vmmeter-action@master
+      - uses: alexellis/setup-arkade@master
+      - name: Install vmmeter
+         run: |
```

Comment:
I think you'll need to un-indent the run to be in the same line as name. Apologies, the suggestion seems to have messed this up.

Reply:
Ah. Whitespace sensitive structures are irritating :)

Reply:
Apologies. That's what I get for using GitHub's in-browser editor for this.